pkg/kubebench/scanner.go

```diff
@@ -23,41 +25,44 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-const (
-	kubeBenchContainerName = "kube-bench"
-)
+// Plugin defines the interface between Starboard and Kubernetes configuration
+// checker with CIS Kubernetes Benchmarks.
+type Plugin interface {
```

Comment:
Even though there's (and will be) only one implementation that relies on Kube-Bench, the purpose of this interface is to share the logic between CLI and the operator.